src/api/todos.ts

```diff
@@ -0,0 +1,19 @@
+import { Todo } from '../types/Todo';
+import { NewTodo } from '../types/NewTodo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const addTodos = (userId: number, data: NewTodo) => {
+  return client.post<Todo[]>(`/todos?userId=${userId}`, data);
+};
+
+export const patchTodos = (todoId: number, data: NewTodo) => {
+  return client.post<Todo[]>(`/todos/${todoId}`, data);
+};
+
+export const deleteTodos = (todoId: number) => {
```

Comment:
The function name 'deleteTodos' should be 'deleteTodo' as it only deletes a single todo.